### ISSUE TITLE
Missing Glottocodes serialized as "NA" in JSON data

### ISSUE THREAD

**Issue by xrotwang** [autotyp/autotyp-data#36]
Missing Glottocodes are sometimes (always?) serialized as string `"NA"`, rather than the semantically correct `null`. E.g.
```shell
$ grep "NA" raw/autotyp-data/data/json/Morphology/VerbSynthesis.json 
    "Glottocode": "NA",
```

**Comment by xrotwang** [comment]
Other than for Glottocodes, that seems to only happen for `MorphemeClasses` -> `Notes`:
```shell
$ grep "\"NA" raw/autotyp-data/data/json/*/*.json | grep -v Glottocode
raw/autotyp-data/data/json/Morphology/MorphemeClasses.json:    "Notes": "NA"
raw/autotyp-data/data/json/Morphology/MorphemeClasses.json:    "Notes": "NA"
raw/autotyp-data/data/json/Morphology/MorphemeClasses.json:    "Notes": "NA"
...
```

**Comment by tzakharko** [comment]
Fixed in v1.0.1
